AIAT-151 Kort administrator credentials (#16)
ScottLogic/kort: README.md

A web application supporting multiple user experience (UX) research methods.

> - [Card Sorting](https://en.wikipedia.org/wiki/Card_sorting)

Reviewer: These have changed to - from *, Did we expect this to happen or is this linting shenanigans?

Author: Linting shenanigans. I'll revert

1. You can connect via [http://localhost:3000](http://localhost:3000)

> 2. The `adminUser` and `adminPassword` variables in `app.js` is the username and password for the account that will be created upon first launch. You can set these variables in the `.env` file or directly in the `docker-compose.yml` file. If you choose to set them as environment variables, make sure to remove the reference to `.env` in `docker-compose.yml`. **Please note that if you define these variables using both methods, the environment variables will take precedence.**

Reviewer 2: I am not 100% sure it is necessary to tell them to remove the reference to `.env`
```suggestion
2. The `adminUser` and `adminPassword` variables in `app.js` is the username and password for the account that will be created upon first launch. You can set these variables in the `.env` file or as an environment variables. **Please note that if you define these variables using both methods, the environment variables will take precedence.**
```
